resources/deprecations.properties

```diff
@@ -1,39 +1,22 @@
 # This file contains deprecated plugins (keys) and the URL the deprecation notice on plugins.jenkins.io and in Jenkins
 # should link to (value). Use this file to mark a plugin as deprecated while continuing to distribute it. If a plugin
 # should be removed from distribution entirely, instead set a deprecation notice URL in artifact-ignores.properties.
-# Shorten URLs if and only if the URLs are to github.com (git.io) or wiki.jenkins.io. Record the real URL in a comment.
 
-# https://github.com/jenkinsci/jenkins/pull/5320
-BlameSubversion = https://git.io/Jn6Co
+BlameSubversion = https://github.com/jenkinsci/jenkins/pull/5320
 # https://github.com/jenkins-infra/update-center2/pull/521
```

Comment:
```suggestion
```

Not directly related, but while we're here 😃

The referenced PR is related to `blitz_io-jenkins` which has since been suspended (improper cleanup in 1bb5edc5925603baaead36f0cde3bbc710d9fa5a).